doc/dev/README.md

```diff
@@ -61,4 +61,4 @@ Some key concepts:
 
 ### Manifest formats
 
-See `ComponentManifest` and `VersionManifest` data structures in [component.go](pkg/repository/component.go) and [version.go](pkg/repository/version.go).
+See `ComponentManifest` and `VersionManifest` data structures in [component.go](../../server/handler/component.go) and [version.go](../../pkg/version/version.go).
```

Comment:
This line seems out-dated, the manifest structures are defined in `pkg/repository/types.go`.